appveyor.yml

```diff
@@ -2,7 +2,12 @@ version: 1.0.{build}
 
 image: 
   - Visual Studio 2022
-  - Ubuntu
+  - Ubuntu2204
+
+init:
+  - sh: |
```

Comment:
1. `sh:`

https://www.appveyor.com/docs/getting-started-with-appveyor-for-linux/#bash-and-powershell
> To run Bash command either put it with `sh:` prefix

2. `|`

One of the [many ways YAML does multi-line strings](https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines) (literal style). Didn't think too much about this, so happy to change this to folded style.